packages/lesswrong/components/ultraFeed/settingsComponents/UltraFeedSettingsComponents.tsx

```diff
@@ -487,59 +641,171 @@ export const AdvancedTruncationSettings: React.FC<AdvancedTruncationSettingsProp
+            value={clampedExploreBiasValue} // Input displays 0-2 range
+            onChange={(e) => {
+              const val = parseFloat(e.target.value);
+              if (!isNaN(val)) {
+                onExploreBiasChange(Math.max(0, Math.min(2, val)));
+              }
+            }}
+            min={0}
+            max={2}
+            step={0.5}
+          />
+        </div>
+        <div className={classes.sourceWeightDescription}>
+          <ul>
+            <li>Left: Strongly prefer content related to what you've read, upvoted, commented on, or from authors you follow.</li>
+            <li>Right: Indifferent to your past interactions, recommend what generally seems good.</li>
+          </ul>
+          <p>
+            This adjusts <code>Log Impact Factor</code> and <code>Subscribed Author Boost</code> located in Advanced Settings and <code>Subscribed Author</code> proportion in Source Weights.
+          </p>
+        </div>
+      </div>
+    </CollapsibleSettingGroup>
+  );
+};
 interface MultipliersSettingsProps {
-  quickTakeBoost: {
-    value: number;
-    error?: string;
-    onChange: (value: number | string) => void;
-  };
-  seenPenalty: {
-    value: number | '';
-    error?: string;
-    onChange: (value: number | string) => void;
-  };
+  formValues: CommentScoringFormState;
+  errors: ZodFormattedError<CommentScoringFormState, string> | null;
+  onFieldChange: (field: keyof CommentScoringFormState, value: number | string) => void;
+  defaultOpen?: boolean;
 }
 
 export const MultipliersSettings: React.FC<MultipliersSettingsProps> = ({
-  quickTakeBoost,
-  seenPenalty,
+  formValues, 
+  errors,
+  onFieldChange,
+  defaultOpen,
 }) => {
   const classes = useStyles(styles);
+
+  const defaultCommentScoringSettings = DEFAULT_SETTINGS.resolverSettings.commentScoring;
+
+  const quickTakeBoostSliderValue = typeof formValues.quickTakeBoost === 'number' ? formValues.quickTakeBoost : defaultCommentScoringSettings.quickTakeBoost;
+  const subscribedAuthorSliderValue = typeof formValues.commentSubscribedAuthorMultiplier === 'number' ? formValues.commentSubscribedAuthorMultiplier : defaultCommentScoringSettings.commentSubscribedAuthorMultiplier;
+  const seenPenaltySliderValue = typeof formValues.ultraFeedSeenPenalty === 'number' ? formValues.ultraFeedSeenPenalty : defaultCommentScoringSettings.ultraFeedSeenPenalty;
+  const commentDecayFactorSliderValue = typeof formValues.commentDecayFactor === 'number' ? formValues.commentDecayFactor : defaultCommentScoringSettings.commentDecayFactor;
+  const commentDecayBiasHoursSliderValue = typeof formValues.commentDecayBiasHours === 'number' ? formValues.commentDecayBiasHours : defaultCommentScoringSettings.commentDecayBiasHours;
+  const threadScoreFirstNSliderValue = typeof formValues.threadScoreFirstN === 'number' ? formValues.threadScoreFirstN : defaultCommentScoringSettings.threadScoreFirstN;
+
+  const quickTakeBoost = {
```

Comment:
These can clearly be abstracted, no?  I'm not really even sure what you need the wrappers for - just the `onChange`?

```
function onChangeForField(fieldName: keyof CommentScoringFormState) {
  return (val: number | string) => onFieldChange(fieldName, val);
}

...

onChangeForField('quickTakeBoost')
```
or something?
```